src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,29 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<String> set = new TreeSet<>();
+        try {
+            String line = Files.readString(Paths.get(fileName));
+            String[] lineList = line
+                    .toLowerCase()
+                    .replaceAll("[^a-z]", "")
+                    .split("");
```

Comment:
```suggestion
                    .toCharArray();
```
буде не так затратно по ресурсах